dotnet/src/Functions/Functions.UnitTests/OpenApi/OpenApiDocumentParserV30FeatureTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.IO;
+using System.Text.Json.Nodes;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Plugins.OpenApi;
+using SemanticKernel.Functions.UnitTests.OpenApi.TestPlugins;
+using Xunit;
+
+namespace SemanticKernel.Functions.UnitTests.OpenApi;
+
+public static class OpenApiDocumentParserV30FeatureTests
+{
+    public class OneOfTests
```

Comment:
I plan to add more test cases for each of the classes. I thought this would help with organization later. But I can merge the methods if that's the preferred way.